apps/marketplace/src/app/artist/[artistId]/page.tsx

```diff
@@ -63,16 +46,16 @@ const Artist: FunctionComponent<ArtistProps> = ({ params }) => {
 
           <SimilarArtists />
         </Container>
-
-        <ProfileModal
-          isOpen={ isAboutModalOpen }
-          name={ `${mockArtist.firstName} ${mockArtist.lastName}` }
-          socials={ socials }
-          onClose={ () => setIsAboutModalOpen(false) }
-        >
-          <Typography variant="subtitle1">{ mockArtist.description }</Typography>
-        </ProfileModal>
       </Stack>
+
+      <ProfileModal
```

Comment:
The Modal is off Center, not sure if this on purpose or from certain flex settings, Didn't look at the full impact in Studio but it did center by adding `justifySelf: "center"` to `sx` in the `Modal.tsx` component.

https://github.com/projectNEWM/newm-web/blob/653f3c5f740edac42c22d9fda8c3079b9073f9ec/packages/elements/src/lib/Modal.tsx#L39